homeassistant/components/switch/homekit_controller.py

```diff
@@ -62,3 +70,20 @@ def turn_off(self, **kwargs):
                             'iid': self._chars['on'],
                             'value': False}]
         self.put_characteristics(characteristics)
+
+    @property
+    def state_attributes(self):
+        """Return the optional state attributes."""
+        data = {}
+
+        for prop, attr in PROP_TO_ATTR.items():
+            value = getattr(self, prop)
+            if value is not None:
+                data[attr] = value
+
+        return data
+
+    @property
+    def outlet_in_use(self):
```

Comment:
We don't need a property. Just assign to a regular instance attribute.

Reply:
ok